README.md

```diff
@@ -78,6 +78,6 @@ The application can be installed as a global npm package and run as a standalone
 To install it and to run web application:
 
 ```sh
-npm install -g autocannon-ui-web
-autocannon-ui-web
+npm install -g autocannon-ui
+autocannon-ui
```

Comment:
let's change this to show how to run it with npx alone. if it works with npx, there's probably not much value in installing it globally